torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -316,8 +316,22 @@ def _create_runtime_wrapper(
             make_output_handler(info, runtime_metadata, trace_joint)
             for info in runtime_metadata.output_info
         )
+    def record_runtime_wrapper_prologue_enter() -> AbstractContextManager[None]:
+        cm: AbstractContextManager[None] = (
+            torch._C._profiler._RecordFunctionFast("AOTDispatcher Runtime Wrapper Prologue")
+            if torch.autograd.profiler._is_profiler_enabled
```

Comment:
@anijain2305 I saw that you have an additional `config.record_pre_graph_bytecode_in_traces` that you branch on in your case. Did you add that because you're worried about the extra overhead even in this case where profiling is off? https://github.com/pytorch/pytorch/pull/154769/files#diff-0bb1756725c4426408938314b0c9d3988ae5bf49994892d7038ad7746e209e9fR617